src/index.html

```diff
@@ -4,14 +4,207 @@
+            minlength="3"
+            maxlength="20"
+          >
+        </div>
+
+        <div class="input">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+            tabindex="2"
+          >
+        </div>
+
+        <div class="input">
+          <label for="how">How old are You?</label>
+          <input
+            type="number"
+            id="how"
+            name="how"
+            tabindex="3"
+            value="12"
+            min="12"
+            max="100"
+          >
+        </div>
+
+        <div class="input">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            id="date"
+            name="date"
+            tabindex="4"
+          >
+        </div>
+
+        <div class="input">
+          <label for="iAccept">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="iAccept"
+            name="iAccept"
+            tabindex="5"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="input">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            tabindex="6"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="input">
+          <label for="pass">Password:</label>
+          <input
+            type="password"
+            id="pass"
+            name="pass"
+            tabindex="7"
+            required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="input">
+          Do you love cats?
+          <input
+            type="radio"
+            id="catsY"
+            name="cats"
+            value="y"
+            tabindex="8"
+          >
+          <label for="catsY">Yes</label>
+          <input
+            type="radio"
+            id="catsN"
+            name="cats"
+            value="n"
+            tabindex="9"
+          >
+          <label for="catsN">No</label>
+        </div>
+
+        <div class="input">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+            name="color"
+            tabindex="10"
+          >
+        </div>
+
+        <div class="input">
+          <label for="timeGoBed">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="timeGoBed"
+            name="tomeGoBed"
+            tabindex="11"
+          >
+        </div>
+
+        <div class="input">
+          <label for="Car">What are your favorite brand of cars?</label>
+          <select
+            name="Car"
```

Comment:
4th item of [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), name attr should use camelCase
```suggestion
            name="car"
```